archetypes/Exchange/Summary/index.tsx

```diff
@@ -20,26 +20,28 @@ type SummaryProps = {
     isLong: boolean;
     commitAction: string;
     receiveIn: number;
+    inputAmount: number;
+    mintGasFee: any;
 };
 
-// const Summary
-export default (({ pool, showBreakdown, amount, isLong, commitAction, receiveIn }) => {
+export default (({ pool, showBreakdown, amount, isLong, receiveIn, inputAmount, commitAction, mintGasFee }) => {
     const [showTransactionDetails, setShowTransactionDetails] = useState(false);
 
     const token = useMemo(() => (isLong ? pool.longToken : pool.shortToken), [isLong, pool.longToken, pool.shortToken]);
-
     const nextPoolState = useMemo(() => pool.getNextPoolState(), [pool.lastPrice]);
-
     const tokenPrice = useMemo(() => (isLong ? pool.getNextLongTokenPrice() : pool.getNextShortTokenPrice()), [isLong]);
 
-    const totalCommitmentAmount = 2000;
-    const totalGasFee = 1.78;
-    const totalCost = totalCommitmentAmount + totalGasFee;
-    const expectedAmount = amount.div(tokenPrice ?? 1).toFixed(3);
+    const totalCommitmentAmount = inputAmount ? toApproxCurrency(inputAmount) : 0;
+    const totalGasFee = mintGasFee;
+    const totalCost = toApproxCurrency(inputAmount);
+    const expectedAmount = amount.div(tokenPrice ?? 1).toFixed(0);
     const expectedPrice = ` at ${toApproxCurrency(tokenPrice ?? 1, 2)} USD/token`;
-    const expectedTokensMinted = `${expectedAmount} ${token.name}`;
-    // const commitAmount = totalCommitmentAmount;
+    const expectedTokensMinted = `${Number(expectedAmount) > 0 ? expectedAmount : ''} ${token.name}`;
     const poolPowerLeverage = pool.leverage;
+    const selectedToken = pool?.name?.split('-')[1]?.split('/')[0];
+    const selectedTokenOraclePrice = toApproxCurrency(pool.oraclePrice);
+    const equivalentExposureBTC = (inputAmount / pool.oraclePrice.toNumber()) * poolPowerLeverage;
+    const commitAmountBTC = inputAmount / pool.oraclePrice.toNumber();
```

Comment:
These wont always be amounts in BTC, could change to just `equivalentExposure` and `commitAmount` or am I thinking of the BTC wrong